### DISCUSSION TITLE
bookmark webpage cuts information

### DISCUSSION THREAD

**Discussion by progsec** [nextcloud/bookmarks#1824] [Q&A]
Hi all,
I "just" started using bookmarks and don't know whether it's a bug or just a missed configuration setting from my side..
I've got b "bigger" bookmark file to sync, and the sync lasts very long (it just display "I'm still working" see hardcopy "running-forever.jpg".
Trying to see if there are further bookmarks added, "sometimes" I see the number of bookmarks, sometimes the numbers are shortened, cut, see attached hardcopy "cut-numbers.jpg"...
Btw.: I use firefox 100.x..
... since I can't change the width of the columns
![running-forever](https://user-images.githubusercontent.com/106964907/172149147-595aa0e5-c617-4845-b5d2-3c3bd8cce398.jpg)
![cut-numbers](https://user-images.githubusercontent.com/106964907/172149156-4a56c8ee-9b03-46c7-89a4-8c7fb97ada56.jpg)
, I don't know what to do.

any ideas?
best regards
jürgen

**Comment by marcelklehr** [comment]
North of 1k bookmarks things can take a while on initial sync. The numbers are already fixed in the development version, just needs a release :)

**Reply by progsec** [reply]
thanks, Mark, for the quick hint!!
;-) so I'll wait, in both cases, "sync" and "numbers" ;-)

as far as I know the number of bookmarks is much higher ...
(collected bookmarks for years, never re-validated, ..)
best regards
jürgen

**Reply by progsec** [reply]
.. a (hopefully) last question:
(I use a mac laptop as source, a raspi as nextcloud server.., maybe I have to shudown the one or the other..)
it still lasts > 18h.
--> can I interupt and restart the sync later?
in case.. does it (re-)start from scratch or does it continue ?
best regards
jürgen

**Reply by marcelklehr** [reply]
Interrupting a sync run is probably ok as long as you have backups

**Reply by progsec** [reply]
great!!! thanks!

**Reply by marcelklehr** [reply]
fyi, the latest release of the bookmarks app should be faster for syncing

**Comment by progsec** [comment]
Hi Marcel,
thanks for the info!
-> .. I installed it.. 

;-) the format of the number of bookmarks has been changed, so now I can ~see, if sync goes forward..
and, as you mentioned, it seems to be faster..

best regards
jürgen
